.github/workflows/Build-And-Test.yml

```diff
@@ -150,7 +150,7 @@ jobs:
         path: ${{ github.workspace }}/bin/DebugAdapterProtocolTests/Debug/CppTests/results.trx
 
   osx_build:
-    runs-on: macos-12
+    runs-on: macos-latest
```

Comment:
>latest [](http://example.com/codeflow?start=19&length=6)

Should we pick one of the version number pools instead so this doesn't immediately swap over?

Reply:
Eh. I guess that is okay.